src/App.tsx

```diff
@@ -1,22 +1,15 @@
 import React, { useEffect } from "react";
 import AppRouter from "./AppRouter";
-import { MaterialUISwitch } from "./components/styled/Switch";
 import { ThemeProvider } from "@mui/material";
 import { createCustomTheme } from "./theme/theme";
 import { GlobalStyle } from "./components/styled/styles";
-import { Box, IconButton } from "@mui/material";
-import { LanguageProvider, useLanguage } from "./components/LanguageContext";
+import { LanguageProvider} from "./components/LanguageContext";
 
-import uaFlag from "./img/ua.png";
-import enFlag from "./img/en.png";
 import { useThemeCustom } from "./theme/ThemeContext";
 
 const App: React.FC = () => {
-  const { darkMode, toggleDarkMode } = useThemeCustom();
-  const { language, toggleLanguage } = useLanguage();
-
+  const { darkMode } = useThemeCustom();
```

Comment:
our cards are working, maybe try to clear the local storage, everything fell into place with this action

Reply:
https://github.com/proga-react-practice/alpha-team-project/assets/144446009/f908833c-9b85-4e41-bdfb-8a687cd66a18

Reply:
I don't understand the second point. If I delete a song from the user, it gets deleted. If I delete a song from the songs, it also gets deleted. Should the song be removed from the songs block when deleted from the user or vice versa? Initially, I wanted the user's data to be saved separately, so that the playlist would make sense in terms of preserving song information, and when a song is deleted from the songs block, it wouldn't be deleted from the user to make it similar to a playlist. What should I do?

Reply:
Okay then, if it is meant to be like that, we can leave as it is

So, the only critical things that I see for now are header height and the whole layout of Cards page with overlapping cards on the search box

thanks for fixing the Playlist cards, now it looks nice